src/include/stir/TimeFrameDefinitions.h

```diff
@@ -102,6 +102,12 @@ class TimeFrameDefinitions
    */
     unsigned int get_time_frame_num(const double start_time, const double end_time) const;
 
+    //! Set number of time frames
+    void set_num_time_frames(int num_time_frames) { frame_times.resize(num_time_frames); }
+
+    //! Set time frame
+    void set_time_frame(const int &frame_num, const double &start, const double &end);
```

Comment:
Passing primitives as references is actually slower. 
Do they need to pass as references?